src/Solnet.Programs/Models/TokenData.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Solnet.Wallet;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Solnet.Programs.Utilities;
+
+namespace Solnet.Programs.Models
+{
+    public class TokenData
```

Comment:
No comments here?

Reply:
Fixed and moved file to folder with remaining NameServiceClient related models.